src/css/icon.css

```diff
@@ -0,0 +1,102 @@
+.g3-icon {
+  display: inline-block;
+  background-repeat: no-repeat;
+  background-position: left top;
+  width: 18px;
+  height: 18px;
+  background-size: 18px 18px;
+}
+
+.g3-icon--lg {
+  width: 40px;
+  height: 40px;
+  background-size: 40px 40px;
+}
+
+.g3-icon--md {
+  width: 18px;
+  height: 18px;
+  background-size: 18px 18px;
+}
+
+.g3-icon--sm {
+  width: 12px;
+  height: 12px;
+  background-size: 12px 12px;
+}
+
+.g3-icon--analysis {
```

Comment:
so the idea with icons is to use them as div background images? I'm wondering if we should rethink how we do icons in general

Reply:
Yes! Let's have more discussion about icons! 💪